### PR TITLE
Exam Submission

### PR DESCRIPTION
**Pull request by geophpherie**

### CODE REVIEW

File: student-work/jeff_beyer/exam/exam.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    >>> read_exam('invalid_file.csv')
+    ('', '', '')
+    '''
+    try:
+        # Read the csv as a data frame
+        df = pd.read_csv(csv_location)
+        # Pull the desired columns
+        info = (list(df['question number']), list(df['question']),
+                list(df['answer']))
+    except:
```
Comment (hobson) [inline student-work/jeff_beyer/exam/exam.py:27]: consider using `except IOError:`

File: student-work/jeff_beyer/exam/exam.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    exam_info = read_exam('https://raw.githubusercontent.com/hackoregon/' +
+                          'civicu-pythonii-summer-2017/ceasar/lessons/' +
+                          'shared-resources/exam.csv')
+    # Read the exam file from the local directory
+    # exam_info = read_exam('exam.csv')
+    # Read the exam file (without input)
+    # exam_info = read_exam()
+
+    # Store the exam info as global variables
+    globals()['QUESTIONS'] = exam_info[0]
```
Comment (hobson) [inline student-work/jeff_beyer/exam/exam.py:56]: Nice! Love the use of `globals()` and reuse of your `read_exam` function. FYI a neat python pattern is to assign a tuple to 2 or more variables all at once (you just must have the same number of assigned variables as the length of the tuple), like this:

`globals()['QUESTIONS'], globals()['QUESTION_TEXTS'], globals()['ANSWERS'] = exam_info`

**Comment by hobson** [comment]
It's fine to copy your entire Django project into the student-work folder, but all I needed was a URL to your repo or a working deployed application so I could verify that it runs.

**Comment by geophpherie** [comment]
@hobson Not sure that I meant to do that. My full repo is at https://github.com/jbeyer16/my_civicu_app

**Comment by hobson** [comment]
Thank you for the link Jeff. Yea `git` magic strikes again. Great work on
your Django app and the exam.

--Hobson

**Pull request merged**